src/options/popup.js

```diff
@@ -196,5 +247,11 @@ export default {
   // If this is defined, then it is used as the inner html instead of layout. This allows for ultimate customisation.
   // Be sure to use the classes `cc-btn` and `cc-ALLOW`, `cc-DENY` or `cc-DISMISS`. They enable the app to register click
   // handlers. You can use other pre-existing classes too. See `src/styles` folder.
-  overrideHTML: null
+  overrideHTML: null,
+
+  // element id that is used to open consent pop-up to review and change user consent
+  consentSettingsElemId: null,
+
+  // categories to be shown and used - if empty all are displayed
+  showCategories: []
```

Comment:
Can we just use the full list as the default? No need for if (empty) -> all cases then.